src/BrandwatchReactAuth.js

```diff
@@ -0,0 +1,61 @@
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import jwtDecode from 'jwt-decode';
+import TokenStore from 'viziaauth';
+
+export default class BrandwatchReactAuth extends Component {
+  getChildContext() {
+    return {
+      brandwatchAuthLogout: this.handleLogout,
```

Comment:
Could we consider putting a reference to `this.store.getToken.bind(this.store, {aud: this.props.audience})` in here so child components can have access to it? might negate the need for `onAuthenticated`?